src/sparseml/pytorch/torchvision/train.py

```diff
@@ -217,15 +222,11 @@ def load_data(traindir, valdir, args):
         print(f"Loading dataset_test from {cache_path}")
         dataset_test, _ = torch.load(cache_path)
     else:
-        if args.weights and args.test_only:
-            weights = torchvision.models.get_weight(args.weights)
-            preprocessing = weights.transforms()
```

Comment:
Is this okay to delete? This is something that isn't in existing image_classification training scripts, but seems like a good feature in torchvision (though this requires 0.13.0+ I believe)

Reply:
fine for now - why in this PR?

Reply:
Just noticed it - haven't hit that branch yet during my simple test runs so far